tests/foreman/ui/test_hostgroup.py

```diff
@@ -382,3 +382,47 @@ def test_positive_nested_host_group_compute_resource(
         )
         child_hostgroup_values = session.hostgroup.read(f'{parent_hg_name}/{child_hg_name}')
         assert child_hostgroup_values['host_group']['deploy'] == cr_bare_metal
+
+
+@pytest.mark.rhel_ver_match('9')
```

Comment:
If you are targetting a single version and your only concern is for it to be supported, why not picking the last one from the list dynamically?

```suggestion
@pytest.mark.rhel_ver_match(str(settings.supportability.content_hosts.rhel.versions[-1]))
```
Such approach could save future generations from test not being executed anymore in case the hardcoded version is being pushed out of the supported list due to EOL

Reply:
Hey, good to see community members doing reviews :)  These days there is  `@pytest.mark.rhel_ver_list([settings.content_host.default_rhel_version])` setting for this, so that the version is changed at a single place once in a while. With the above approach is you may pick the `_fips` variant, which might not be a problem or not

Reply:
Updated with @pondrejk 's solution.